File: sum.py
from telnetlib import STATUS


print("This program adds two numbers!")
num1 = input("The first number is ")
num2 = input("The second number is ")
sum = int(num1) + int(num2)
print("The sum of " + num1 + " and " + num2 + " is " + str(sum) + "!")

File: sum.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int num1;
    int num2;

    cout << "This program adds two numbers!" << endl;
    cout << "The first number is ";

    cin >> num1;

    cout << "The second number is ";

    cin >> num2;

    int sum = num1 + num2;
    cout << "The sum of " << num1 << " and " << num2 << " is " << sum << "!" << endl;

    return 0;
}